test/hotspot/jtreg/compiler/vectorization/TestFloatConversionsVectorNaN.java

```diff
@@ -47,16 +50,30 @@ public class TestFloatConversionsVectorNaN {
     private static float  [] fout;
 
     public static void main(String args[]) {
-        TestFramework.runWithFlags("-XX:-TieredCompilation",
-                                   "-XX:CompileThresholdScaling=0.3");
+        TestFramework framework = new TestFramework(TestFloatConversionsVectorNaN.class);
```

Comment:
Is it more usual to take this approach or to call `TestFramework.runWithFlags` multiple times?

Reply:
Not quite sure, but seems the similar tests changes are following the same pattern.